plutus-tx-plugin/test/Plugin/Errors/Spec.hs

```diff
@@ -71,8 +71,8 @@ mutualRecursionUnfoldingsLocal = plc (Proxy @"mutualRecursionUnfoldingsLocal") (
 literalCaseInt :: CompiledCode (Integer -> Integer)
 literalCaseInt = plc (Proxy @"literalCaseInt") (\case { 1 -> 2; x -> x})
 
-literalCaseBs :: CompiledCode (Builtins.ByteString -> Builtins.ByteString)
-literalCaseBs = plc (Proxy @"literalCaseBs") (\x -> case x of { "abc" -> ""; x -> x})
```

Comment:
I actually like having this test that literal patterns  don't work!

Reply:
This code doesn't compile anymore as `Builtins.ByteString` has no `Haskell.Eq` instance. I guess this test is checked by GHC now.

Reply:
That's actually pretty good.

Reply:
Unfortunately had to get it back.